fem/libceed_mass.qf

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) 2010, Lawrence Livermore National Security, LLC. Produced at
```

Comment:
Same thing, should be in the `ceed` subdir.

Reply:
I'll remove this file too.